vsrgtools/freqs.py

```diff
@@ -206,12 +206,15 @@ def expr(
             case MeanMode.LEHMER:
                 p = kwargs.pop("p", self.value)
 
-                expr = ExprList((f"{v} neutral - {eps} + D{i}!" for i, v in enumerate(evars)))
+                if p > 0:
+                    eps = 0
+
+                expr = ExprList((f"{v} neutral - {eps} max D{i}!" for i, v in enumerate(evars)))
```

Comment:
`{eps} max` is certainly wrong for negative inputs, which are valid for `p >= 1`. Including chroma.